code/game/objects/effects/decals/crayon.dm

```diff
@@ -31,4 +32,45 @@
 
 	add_overlay(list(mainOverlay, shadeOverlay))
 
+	SSicon_cache.crayon_cache.Cut()
+
 	add_hiddenprint(usr)
+
+/obj/effect/decal/cleanable/chalk
```

Comment:
why is this all copypasted instead of just being a child of /cleanable/crayon?

Reply:
If I ran it as that, I wouldn't be able to change the references to "crayon" into "chalk" without some refactoring of the cleanable/crayon code.  Do you want me to refactor it?

Reply:
That'd be preferable to copypasta, yeah

Reply:
Refactored!